pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -33,6 +34,10 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
+// Changes outside of the kubernetes universe could cause the approval decision
+// to change. We get around this be frequently resyncing.
```

Comment:
Our use case is sometimes nodes post a CSR before the RBAC role bindings are initialize during cluster start up. In that particular case, 15s would be great. In the more general case, a few minutes seems appropriate.

If there are a lot of CSRs in the system, they will all be reprocessed with each resync, right? Approved/Denied ones will be quickly processed, but they will all go through this queue each cycle, right?